### DISCUSSION TITLE
Trending 🔥 on the welcome page is useless

### DISCUSSION THREAD

**Discussion by ilveropera** [TeamNewPipe/NewPipe#11697] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Given the open-source and privacy-focused nature of **NewPipe**, I believe it would be much more useful to see **recent searches** or videos/music I've listened to on the **welcome page**, instead of the popular videos trending on YouTube.

So, as soon as I open NewPipe, I would see the videos, music, and podcasts I recently searched for, along with the playback progress, similar to YouTube’s history section.

I find those suggestions **misleading** and not useful.

**Comment by Stypox** [comment, marked as answer]
Just change the content of the main page in the settings

**Reply by ilveropera** [reply]
Thanks, if someone find this discussion:
Settings/content/content of main page/+
